spec/services/jwt/renew_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Jwt::Renew do
+  let!(:token) { Jwt::Generate.new(user_id: 1).token }
+  let(:jti) { JWT::decode(token, ENV['JWT_SECRET_TOKEN'], false).first['jti'] }
+  
+  before do
+    memory_store = ActiveSupport::Cache.lookup_store(:memory_store)
+    allow(Rails).to receive(:cache).and_return(memory_store)
+    Rails.cache.clear
+  end
+
+  context '.revoke!' do
+    subject { described_class.new(token).revoke! }
+
+    it 'adds token to jti blacklist' do
+      allow(Rails.cache).to receive(:write)
+      subject
+      expect(Rails.cache).to have_received(:write).with("jwt_jti_denied/#{jti}", true)
```

Comment:
if you read only this spec, can you tell what is being tested, is `subject` descriptive enough?